conda/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "PHG2" %}
+
+package:
+  name: {{ name|lower }}
+  version: "{{ PHG2_VERSION }}"
+
+source:
+  url: https://github.com/maize-genetics/phg_v2/releases/download/{{ PHG2_VERSION }}/PHGv2-v2.2.tar
```

Comment:
So if we update the version to 2.3 does everything but this line update automatically?  Is there a way to get the PHGv2-v2.2.tar also automatically picked up during the release process?

Reply:
Actually I just did it now instead. Please see [this commit](https://github.com/maize-genetics/phg_v2/pull/67/commits/c6c94b8d2f2ccb889269cc2f08fad68c12319886).